conda_forge_tick/auto_tick.xsh

```diff
@@ -121,6 +121,25 @@ def run(attrs, migrator, feedstock=None, protocol='ssh',
     return migrate_return, pr_json
 
 
+def add_rebuild(migrators):
+    """Adds rebuild migrators.
+
+    Parameters
+    ----------
+    migrators : list of Migrator
+        The list of migrators to run.
+
+    """
+
+    pygraph = copy.deepcopy(gx)
+    for node, attrs in gx.node.items():
+        if 'python' not in attrs.get('meta_yaml', {}).get('requirements', {}).get('host', []):
+            pygraph.remove_node(node)
+        if attrs.get('meta_yaml', {}).get('build', {}).get('noarch') == 'python':
+            pygraph.remove_node(node)
+    migrators.append(Rebuild(graph=pygraph, pr_limit=1, name='Python 3.7'))
```

Comment:
Would it be possible to handle this differently? In this upcoming round we may have to do python 3.7, compilers, and R 3.5.1. Would it be possible to build the subgraphs for each of these migrations separately and then use [compose](https://networkx.github.io/documentation/stable/reference/algorithms/generated/networkx.algorithms.operators.binary.compose.html?highlight=compose#networkx.algorithms.operators.binary.compose) (I think there is a `nx.compose` but I couldn't find it in the docs quickly). The main benefit of this is that as new migrations roll in we can `compose` the new subgraph in. I guess we can also "compose" the name of the migration by `' '.join(migration_names)`?